OpenTabletDriver.UX/Controls/PluginSettingStoreEditor.cs

```diff
@@ -71,12 +72,14 @@ private IEnumerable<Control> GetControlsForStore(PluginSettingStore store)
             if (store != null)
             {
                 var type = store.GetTypeInfo<TSource>();
-                return GetControlsForType(store, type);
-            }
-            else
-            {
-                return Array.Empty<Control>();
+
+                if (type != null)
+                    return GetControlsForType(store, type);
+                else
+                    Log.Write("PluginSettingStoreEditor", $"Failed to get type info. {(store.Path == null ? "(store path is null)" : "")}", LogLevel.Error);
```

Comment:
This reads slightly confusing - I would extract the `store.Path` stuff into a variable to better indicate intention (that you only want to tell when path is null), otherwise this reads like you meant to do `store.Path == null ? "fallback" : store.Path`.

```suggestion
                    var isStorePathNull = store.Path == null ? "(store path is null)" : "";
                    Log.Write("PluginSettingStoreEditor", $"Failed to get type info. {isStorePathNull}", LogLevel.Error);
```